src/common/logger.go

```diff
@@ -0,0 +1,44 @@
+package common
+
+import (
+	"fmt"
+	"log/slog"
+	"os"
+	"path"
+
+	"github.com/urfave/cli/v2"
+)
+
+// the current slog does not support enable/disable feature, might need a wrap-arround struct
+var CgTopLogger slog.Logger
+
+func LoadLoggers() error {
+	if Conf.Trace.Enable_JSON == true {
+		level := new(slog.LevelVar)
+		if Conf.Trace.Cgroups == "INFO" {
```

Comment:
Please create a level string parsing function.  It should not be case sensitive.

"" and "off" should mean it's off.  Level is just an int, so we can use LevelError+1 for off, since no logs will be high enough threshold.